language/move-binary-format/src/cursor.rs

```diff
@@ -0,0 +1,153 @@
+use core::cmp;
```

Comment:
Just for clarity, here could be a comment that the code is not ours but is copied from the std library with some tweaks.

Reply:
Done